src/components/productSummary/ProductSummary.jsx

```diff
@@ -0,0 +1,46 @@
+import React from "react";
+import "./ProductSummary.css";
+
+const ProductSummary = ({
+  accommodation,
+  address,
+  guests,
+  bedrooms,
+  beds,
+  baths,
+  starGrade,
+  reviews,
+}) => {
+  return (
+    <div className="product-summary-container">
+      <h2 className="address">
+        {accommodation} in {address}
+      </h2>
+      <div>
+        <span>{guests} guests</span> <span>&#183;</span>{" "}
+        <span>{bedrooms} bedrooms</span> <span>&#183;</span>
+        <span> {beds} beds</span> <span>&#183;</span> <span>{baths} baths</span>
```

Comment:
It would be great to add a conditional for singular and plural. 

For example: 
1 guest --> 2 **guests**
1 bedroom --> 3 **bedrooms**